assets/js/frontend/address-i18n.js

```diff
@@ -104,7 +104,9 @@ jQuery( function( $ ) {
 
 				// Class changes.
 				field.removeClass( 'form-row-first form-row-last form-row-wide' );
```

Comment:
I would suggest to include this in the if statement too

Reply:
I have included this line